Detectors/MUON/MCH/Workflow/src/digits-reader-tb-workflow.cxx

```diff
@@ -0,0 +1,45 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runFileReader.cxx
+/// \author  Andrea Ferrero
+///
+/// \brief This is an executable that reads a data file from disk and sends the data to QC via DPL.
```

Comment:
same remark as above : update to reflect current file

Reply:
Done.